bluepysnap/edges.py

```diff
@@ -390,6 +400,16 @@ def _nodes(self, population_name):
         result = self._edge_storage.circuit.nodes[population_name]
         return result
 
+    @property
+    def config(self):
+        """Population config dictionary."""
+        return self._config
+
+    @property
+    def type(self):
+        """Population type."""
+        return self.config.get('type', DEFAULT_EDGE_TYPE)
```

Comment:
My question was whether we should use the `DEFAULT_EDGE_TYPE` here if `type` is not given or get it from the h5 files but I just realized that edges don't have `model_type` in them so the only way to get it right would be to deduce it by using the class definitions in `bbp.py`

Reply:
I think the current implementation is fine. Class definitions from `bbp.py` can be used if you want to validate the value of `self.config['type']`. Do we want to validate it? I am not sure.

Reply:
Yeah, on the other hand, our main point is to provide API to access the data and it is users' responsibility to ensure the data is in the expected format. On the other hand though, we kind of maintain the format and it would be nice if users could easily verify that their files and config is what is expected. 

Should it be included in the API? My intuition is no but I also see the logic of shipping them together: Changes in the validation and API would go hand in hand.

Reply:
The current codestyle of the project that such validation is located within `validation` module. Lets keep consistent codestyle.

Reply:
Given that development in circuit building and the addition of new features to the sonata configs could easily lead to inconsistent or deprecated configs, imho validation is crucial to remain in snap.

Reply:
I tend to agree that if a model type can be specified in the config, it should be validated. Even if, currently, that given type is only used as a return value here.